docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx

```diff
@@ -140,23 +140,23 @@ Please follow the instructions below, or in the video.
 </p>
 
 :::caution
-If the station and WG1000 Gateway are more than 50m/164ft apart, you must install the LoRa antenna. Else the station might not be able to properly communicate with WG1000 gateway, resulting in less to none data being shared to the network, and thus reduce WXM rewards.
+If the station and WG1000 Gateway are more than 50m/164ft apart, you must install the RF antenna. Else the station might not be able to properly communicate with WG1000 gateway, resulting in less to none data being shared to the network, and thus reduce WXM rewards.
 :::
 
 <details class="accordion">
 
-<summary>WG1000 Gateway LoRa Antenna Installation Instructions</summary>
+<summary>WG1000 Gateway RF Antenna Installation Instructions</summary>
 
 
 #### WG1000 Gateway Antenna Troubleshooting
 
-WG1000 Gateway is equipped with two antennas, the GPS and the LoRa. The GPS antenna is used to determine the location (coordinates) and altitude of the weather station,
-while the LoRa antenna is responsible for the communication between the WG1000 Gateway and the weather station.
+WG1000 Gateway is equipped with two antennas, the GPS and the RF. The GPS antenna is used to determine the location (coordinates) and altitude of the weather station,
+while the RF antenna is responsible for the communication between the WG1000 Gateway and the weather station.
 
-On the left side of the WG1000 Gateway you may notice that there are two different caps, a grey for GPS and a blue for LoRa.
+On the left side of the WG1000 Gateway you may notice that there are two different caps, a grey for GPS and a blue for RF (LoRa protocol) .
```

Comment:
Based on the description of the task we don't use LoRa protocol on the M5 so I suggest removing the parentheses.

Reply:
Pavlos, it is written on the side of the cap "LORA". Don't you think it will be less confusing if we replace this with "RF"?

Reply:
Hmm yes, but as it is now it implies that LoRa protocol is used, which doesn't. Let's do something like that??

> and a blue for RF (labeled as "LoRa") .

Reply:
You really believe that is better ? if you do I will follow

Reply:
Yes I think it's better to differentiate the label on the cap with the LoRa which we do not use. So please proceed with the following just to be 100% clear:

> and a blue for RF (labeled as "LoRa" in the cap).